### PR TITLE
Add uptime check for known hosts

### PR DESCRIPTION
**Pull request by mariliamelo**
**Release Note**

```release-note
Add uptime checks for known hosts
```

### CODE REVIEW

**Comment by mariliamelo** [comment]
I'm not sure if I'm allowed to create the uptime check and also the alert in the same PR. Let me know if I should separate it.

File: docs/playbooks/alerts/HostDown.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+failed the check for 5min.
+
+## Triage steps
+
+There are multiple components involved here so you need to thoroughly
+check every one of them:
+
+- Check the IP address assigned by Cloud Load Balancer: run `terraform
+  output` and check the `lb_ip` from the result.
+- Run `dig encv.org` (or the host that's down) and ensure the IP address
```
Comment (sethvargo) [inline docs/playbooks/alerts/HostDown.md:17]: The URL could be anything. We should just say "the host url"

File: docs/playbooks/alerts/HostDown.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  matches the one you get above.
+- Check https://status.cloud.google.com/ and see if any related services
+  are having an outage right now.
+- If none of the above steps yield useful insight, try follow
+  http://cloud/load-balancing/docs/https/troubleshooting-ext-https-lbs
```
Comment (sethvargo) [inline docs/playbooks/alerts/HostDown.md:22]: This link wont work externally

Comment (sethvargo) [reply docs/playbooks/alerts/HostDown.md:22]: And should be https

File: terraform/alerting/alerts.tf [hunk 3]
```diff
@@ -50,6 +50,41 @@ resource "google_monitoring_alert_policy" "CloudSchedulerJobFailed" {
...
 
+resource "google_monitoring_alert_policy" "probers" {
+  project = var.project
+
+  display_name = "HostDown"
+  combiner     = "OR"
+  conditions {
+    display_name = "Host is unreachable"
+    condition_monitoring_query_language {
+      duration = "300s"
```
Comment (sethvargo) [inline terraform/alerting/alerts.tf:61]: I think I'd prefer to be alerted sooner if a host is completely down. Can we alert on the 1min interval?

File: terraform/alerting/probers.tf [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    labels = {
+      project_id = var.project
+      host       = each.key
+    }
+  }
+
+  depends_on = [
+    null_resource.manual-step-to-enable-workspace
+  ]
+}
```
Comment (sethvargo) [inline terraform/alerting/probers.tf:42]: Need newline EOF

File: terraform/alerting/probers.tf [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  monitored_resource {
+    type = "uptime_url"
+    labels = {
+      project_id = var.project
+      host       = each.key
+    }
+  }
+
+  depends_on = [
+    null_resource.manual-step-to-enable-workspace
```
Comment (sethvargo) [inline terraform/alerting/probers.tf:40]:
```suggestion
    null_resource.manual-step-to-enable-workspace,
```

**Review by sethvargo — CHANGES_REQUESTED** [review]

**Comment by mariliamelo** [comment]
Part of #1295

**Review by sethvargo — APPROVED** [review]

**Pull request merged**